test/common/http/conn_manager_impl_test.cc

```diff
@@ -787,10 +788,39 @@ TEST_F(HttpConnectionManagerImplTest, StartAndFinishSpanNormalFlow) {
   // No decorator.
   EXPECT_CALL(*route_config_provider_.route_config_->route_, decorator())
       .WillRepeatedly(Return(nullptr));
+  envoy::type::FractionalPercent percent1;
+  percent1.set_numerator(100);
+  envoy::type::FractionalPercent percent2;
+  percent2.set_numerator(10000);
+  percent2.set_denominator(envoy::type::FractionalPercent::TEN_THOUSAND);
+  tracing_config_ = std::make_unique<TracingConnectionManagerConfig>(TracingConnectionManagerConfig{
+      Tracing::OperationName::Ingress,
+      {{Tracing::CustomTagType::LITERAL, "l-tag", "l"},
+       {Tracing::CustomTagType::REQUEST_HEADER, ":method", ":method"},
```

Comment:
For completeness, maybe add a literal and environment that is overridden, and also add an environment that is not overridden.

Reply:
i provide a test case suite, hope it doesn't overkill.